command/agent/cache/lease_cache.go

```diff
@@ -299,6 +299,13 @@ func (c *LeaseCache) Send(ctx context.Context, req *SendRequest) (*SendResponse,
 		RequestPath: req.Request.URL.Path,
 	}
 
+	lastRenewed, err := http.ParseTime(resp.Response.Header.Get("Date"))
+	if err != nil {
+		c.logger.Error("failed to parse response header date", "error", err)
+		return nil, err
+	}
+	index.LastRenewed = lastRenewed
```

Comment:
Brought up offline, but we might want to use `time.Now()`for the initial value as well since the renewal update and calculation going forward uses the OS's time.

Reply:
Done in b3c32c1ec918d083a795b882c656f773b84604ba